### ISSUE TITLE
fine_tune

### ISSUE THREAD

**Issue by NoDojuanwang** [dengyecode/hourglassattention#8]
Thank you very much for your excellent work.
Thanks ！！！
 I would like to ask why epoch starts from 1 when using the "fine tuning" command, instead of starting from epoch after training. Does this result in restarting training?

**Comment by dengyecode** [comment]
I apologize for the inconvenience. The reason for this issue is that we overlooked the problem of epoch recording when initially writing the code. As a result, when fine-tuning the model, the model's logs would start recording from zero, which would overwrite the previous relevant files. There is a relatively simple solution. For example, if you intend to fine-tune the model in the "Paris" folder, you can copy the current "latest_net_G.pth" and "latest_net_D.pth" files in the "Paris" folder to a new folder (let's say the folder is called "Paris_new"). Then, in the "base_options.py" file, you can change the "--name" parameter on line 15 to "Paris_new". By doing this, you will not overwrite the previous training-related files.

**Comment by NoDojuanwang** [comment]
Thanks！Thank you very much for your reply, it has been very helpful to me.
 I would like to ask a related question. The commands for fine-tuning and testing both use "continue_train", but it does not specify the number of training sessions. Can I set the number of fine-tuning and testing sessions in the "_niter" parameter of the "train_options" file.

**Comment by SHIJILIANG01** [comment]
Hello, did you test successfully?

**Comment by dengyecode** [comment]
> _niter

Sorry, “continue_train” is redundant in the test phase and should be removed. Of course in fine_tuning sessions you can determine the epoch by setting "_niter" by.

**Comment by NoDojuanwang** [comment]
Yes, I made changes to the code as soon as cuda out&nbsp; of memory error occurred




------------------&nbsp;Original&nbsp;------------------
From:                                                                                                                        "dengyecode/hourglassattention"                                                                                    ***@***.***&gt;;
Date:&nbsp;Thu, Jun 15, 2023 05:13 PM
***@***.***&gt;;
***@***.******@***.***&gt;;
Subject:&nbsp;Re: [dengyecode/hourglassattention] fine_tune (Issue #8)





 
Hello, did you test successfully?
